compio/benches/fs.rs

```diff
@@ -160,6 +196,34 @@ fn read_all_tokio(b: &mut Bencher, (path, len): &(&Path, u64)) {
     })
 }
 
+fn read_all_tokio_std(b: &mut Bencher, (path, len): &(&Path, u64)) {
+    let runtime = tokio::runtime::Builder::new_current_thread()
+        .enable_all()
+        .build()
+        .unwrap();
+    b.to_async(&runtime).iter_custom(|iter| async move {
+        let mut buffer = [0u8; BUFFER_SIZE];
+
+        let start = Instant::now();
+        for _i in 0..iter {
+            let mut file = std::fs::File::open(path).unwrap();
+            let len = *len;
+            buffer = tokio::task::spawn_blocking(move || {
```

Comment:
In this function, all read operations are performed inside the same `spawn_blocking` closure. Why it's not as fast as `read_all_std`? Because you open the file *inside* the for loop. The open-file operation costs a lot.